src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -182,6 +182,16 @@ impl HtmlHandlebars {
         handlebars.register_helper("previous", Box::new(helpers::navigation::previous));
         handlebars.register_helper("next", Box::new(helpers::navigation::next));
     }
+
+    fn copy_additional_css(&self, book: &MDBook) -> Result<(), Box<Error>> {
```

Comment:
Cheers for picking up on that! I had another look at the names I'm using to make sure they reflect what the function is actually doing. In the end I went with the slightly longer `copy_additional_css_and_js`.